modules/dynamic-plugins/proc-custom-plugin-conversion.adoc

```diff
@@ -0,0 +1,131 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="proc-example-custom-plugin-installation_{context}"]
+= Example of installing a custom plugin in {product}
+This example demonstrates how to package and install dynamic plugins using the {product-custom-resource-type} *Entity Feedback* community plugin that is not included in {product} pre-installed dynamic plugins. 
+
+.Limitations
+* You need to ensure that your custom plugin is built with a compatible version of {product-custom-resource-type}. In {product-short}, click *Settings*. Your custom plugin must be compatible with the *Backstage Version* (or the closest previous version) that is displayed in the *Metadata* section of {product}.
++
+For example, if you view the link:https://github.com/backstage/community-plugins/commits/main/workspaces/entity-feedback/backstage.json[history] of the `backstage.json` file for the *Entity Feedback* plugin, the `1fc87de` commit is closest previous version to {product-custom-resource-type} version of 1.39.1.
++
+.`backstage.json` file history in Github
+image::rhdh/custom-limitations.png[]
+
+
+.Prerequisites
+* Your local environment meets the following requirements:
+
+* *Node.js:* Version 22.x
+* *Yarn:* Version 4.x
+* *git CLI*
+* *jq CLI:* Command-line JSON processor
+* *OpenShift CLI (oc):* The client for interacting with your OpenShift cluster.
+* *Container runtime:* Either podman or docker is required for packaging the plugin into an OCI image and logging into registries.
+* *Container registry access:* Access to an OCI-compliant container registry (such as the internal OpenShift registry or a public registry like Quay.io).
+//* *Command-line tool to build and package plugins:* The `@red-hat-developer-hub/cli` package. For compatibility with the most recent features and fixes, use the latest version (`@latest` tag) version of the  `@red-hat-developer-hub/cli` package.
+
+.Procedure
+. Clone the source code for the *Entity Feedback* plugin, as follows:
++
+[source,terminal]
+----
+git clone https://github.com/backstage/community-plugins.git
+cd community-plugins
```

Comment:
Is the `git checkout` command and text about versions not included for a reason?